src/components/Navigation/SearchDrawer.js

```diff
@@ -245,21 +252,12 @@ function SearchDrawer({ children, active, toggle }) {
                       <Grid container justify="flex-end">
                         <MenuList className={classes.searchResults}>
                           {data.slice(0, 10).map(suggestion => (
-                            <Link
-                              href={`/search?q=${suggestion.value}`}
+                            <MenuItem
                               key={`${suggestion.value}-${suggestion._click_id}`} // eslint-disable-line no-underscore-dangle
                               {...getItemProps({ item: suggestion })}
                             >
-                              <Tooltip
-                                title={suggestion.value}
-                                placement="bottom-start"
-                                classes={{ tooltip: classes.tooltip }}
-                              >
-                                <Typography color="textSecondary" noWrap>
-                                  {suggestion.value}
-                                </Typography>
-                              </Tooltip>
-                            </Link>
+                              {suggestion.label}
```

Comment:
`Tooltip` is still needed for when the suggestion is too long to fit:
![Screenshot from 2020-03-11 15-42-58](https://user-images.githubusercontent.com/1779590/76418107-17ec2500-63af-11ea-8c2f-c3a40235c898.png)